### DISCUSSION TITLE
Multiple generator block?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [prisma/prisma#15170] [Q&A] [answered]
Is it possible to have multiple generator blocks and be able to switch between them? I am using https://prisma-client-py.readthedocs.io/en/stable/ and would like to be able to use the same schema file for both JavaScript and Python, but generate different files.

**Comment by austincrim** [comment, marked as answer]
Hey @tyteen4a03 👋,

It is possible to have multiple `generator` blocks, but Prisma will execute all generators when generating.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
